### PR TITLE
Reduce number of Detekt issues

### PR DESCRIPTION
**Pull request by timkrueger**
PR #1969 is failing because of they are too much of them

### CODE REVIEW

File: app/src/main/java/com/nextcloud/talk/controllers/LocationPickerController.kt [hunk 1]
```diff
@@ -213,7 +213,7 @@ class LocationPickerController(args: Bundle) :
         return true
     }
 
-    @Suppress("Detekt.TooGenericExceptionCaught")
+    @Suppress("Detekt.TooGenericExceptionCaught", "Detekt.ComplexMethod")
```
Comment (AndyScherzinger) [inline app/src/main/java/com/nextcloud/talk/controllers/LocationPickerController.kt:216]: still needed since you refactored the method?

Comment (timkrueger) [reply app/src/main/java/com/nextcloud/talk/controllers/LocationPickerController.kt:216]: Yes. The method is still too complex Detekt says. But I can live with it.

But if you insist I will look again on it :-)

Comment (AndyScherzinger) [reply app/src/main/java/com/nextcloud/talk/controllers/LocationPickerController.kt:216]: Nah, all good then 👍

**Review by AndyScherzinger — APPROVED** [review]

**Pull request merged**